src/points/cardSort.ts

```diff
@@ -0,0 +1,18 @@
+import { Props as ActivityCardProps } from 'src/points/ActivityCard'
+
+export function sortByAmountAndTitle(a: ActivityCardProps, b: ActivityCardProps) {
```

Comment:
nit: i wonder if we should avoid mutations, and therefore have the .sort inside this function rather than on the caller. i think generally we want to avoid mutations to keep things predictable

Reply:
hmm, this function is a comparator, it does not do any sorting. I will update the naming.

when we are actually do sorting in place using this comparator, we are sorting the result of the `.map()` function, which I believe is safe to mutate in that case:
https://github.com/valora-inc/wallet/blob/331b6fc2115e158ffe88316c0f215953c77af776/src/points/ActivityCardSection.tsx#L59